lib/elixir/test/erlang/tokenizer_test.erl

```diff
@@ -209,7 +209,11 @@ capture_test() ->
    {unary_op, {1, 2, nil}, 'not'},
    {int, {1, 6, 1}, "1"},
    {',', {1, 7, 0}},
-   {int, {1, 9, 2}, "2"}] = tokenize("&not 1, 2").
+   {int, {1, 9, 2}, "2"}] = tokenize("&not 1, 2"),
+  [{capture_op, {1, 1, nil}, '&'},
+   {identifier, {1, 2, nil}, '/'},
+   {mult_op, {1, 3, nil}, '/'},
+   {int, {1, 4, 2}, "2"}] = tokenize("&//2").
```

Comment:
I think we also need to make `&/  /2` and `& / /2` work and the current patch does not cover those cases. I am not sure how to make it work though.

Reply:
I think I was able to find a working implementation! Can you please try the suggestion in the comment above? We also need tests in this function for the following constructs: `&//2`, `& / /2`, `& //2`, `&/ /2`, and `& &/1`. Thank you!